set.mm

```diff
@@ -425279,9 +425340,9 @@ afterwards the last vertex (which is the first vertex) itself ("walking
       IYOUUEUURSUUAABCFGHIJKLMNYJVSYKWD $.
 
     ${
-      $d C w $.
-      numclwwlk.t $e |- T = ( w e. ( X C N )
-            |-> <. ( w substr <. 0 , ( N - 2 ) >. ) , ( w ` ( N - 1 ) ) >. ) $.
+      $d C u $.  $d F u $.  $d V u $.
```

Comment:
maybe you can move $d F u $. and  $d V u $. a bit down, where they begin to be necessary.
**Edit**: maybe it's the correct place, I do not see the following $e and $p statements to tell for sure.

Reply:
It is at the correct place.